packages/acp4/acp4.1.0.1/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+authors: "Francois Berenger"
+maintainer: "unixjunkie@sdf.org"
+license: "GPL-3.0-only"
+homepage: "https://github.com/tsudalab/ACP4"
+dev-repo: "git+https://github.com/tsudalab/ACP4.git"
+bug-reports: "https://github.com/UnixJunkie/ACP4/issues"
+build: ["dune" "build" "-p" name "-j" jobs]
+install: [
+  ["cp" "bin/acp4_ph4.py" "%{bin}%/acp4_ph4.py"]
+]
+depends: [
+  "batteries"
+  "bst"
+  "dolog"
```

Comment:
```suggestion
  "dolog" {> "4.0.0"}
```
The library became wrapped with the port to dune. Right now the lower-bound build fails with `missing module Dolog`